### PR TITLE
Adding advisories for apache spark

### PR DESCRIPTION
**Pull request by mamccorm**
Creating advisories for apache spark.
Note, as this is for a new package, will require the following to be merged first: https://github.com/wolfi-dev/os/pull/12309

### CODE REVIEW

File: spark.yaml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+  - id: CVE-2022-3510
+    aliases:
+      - GHSA-4gg5-vx3j-xwc7
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: protobuf-java classes are not included in runtime jars, but its metadata is still listed in META-INF pom.xml and properties files
```
Comment (pdeslaur) [inline spark.yaml:120]: I'm not sure I agree with this - I have found the protobuf classes in the shaded jar

**Review by luhring — CHANGES_REQUESTED** [review]
(placing a hold until we sync next week!)

File: spark.yaml (hunk 1 above)
Comment (mamccorm) [reply spark.yaml]: Resolving as we paired up earlier and walked through some of the items, including this one :)

**Comment by luhring** [comment]
Continuing this in #1112

**Pull request closed without merging**